frontend/src/components/6.Swiping/SwipingPage.css

```diff
@@ -27,6 +27,10 @@
   border-radius: 50%;
 }
 
+.hide {
+  display: none;
```

Comment:
Personally, I think it would be better to change this to `visibility: hidden` so that it doesn't shift elements on the screen.